src/main/java/com/faforever/client/vault/replay/LiveReplayController.java

```diff
@@ -116,4 +131,10 @@ public Node getRoot() {
     return new SimpleStringProperty(Joiner.on(i18n.get("textSeparator")).join(modNames));
   }
 
+  @NotNull
+  private String readableGameStartTime(Instant startTime) {
```

Comment:
I suppose it should go into TimeService? Any preferences regarding naming / format?

Reply:
yeah u should

Reply:
Take something short